toolchest_client/tools/api.py

```diff
@@ -105,6 +105,33 @@ def kraken2(inputs, output_path, tool_args=""):
     instance.run()
 
 
+def STAR(output_path, read_one=None, read_two=None, tool_args=""):
+    """Runs STAR (for alignment) via Toolchest.
+
+    :param tool_args: (optional) Additional arguments to be passed to STAR.
+    :param read_one: (optional) Path to the file containing R1 short reads.
+    :param read_two: (optional) Path to the file containing R2 short reads.
+    :param output_path: Path (client-side) where the output file will be downloaded.
+
+    Usage::
+
+        >>> import toolchest_client as toolchest
+        >>> toolchest.STAR(
+        ...     read_one="./r1.fastq",
+        ...     read_two="./r2.fastq",
+        ...     tool_args="--mode bold",
+        ...     output_path="scratch"
+        ... )
+
+    """
+
+    instance = STAR(
+        tool_args=tool_args,
+        output_name='output.zip',
+        inputs=[read_one, read_two],
```

Comment:
Won't this require an `input_prefix_mapping`? It might break the pattern currently employed, which assumes that each input corresponds to one tool arg

Reply:
It's hard to say since STAR's syntax requires specifying multiple input filepaths under the same tag, so we'd need to expand our current implementation (unless I'm misunderstanding something).

Reply:
Huh, actually looks like it'll be fine. On the worker node, inputs with no prefix are translated to an arg with a value looking like f"{path} " (note the space). So default handling should work with STAR.